bin/hdfcoinc/pycbc_fit_sngls_split_binned

```diff
@@ -46,6 +46,8 @@ parser.add_argument('--output-file', type=str, required=True,
 parser.add_argument('--verbose', action='store_true')
 parser.add_argument('--bin-param', default='template_duration', type=str,
                     help="parameter for on-plot splitting. Default: template_duration")
+parser.add_argument('--bin-param-log', default=False, action='store_true',
```

Comment:
btw, this was the previous behaviour : https://github.com/gwastro/pycbc/blob/v1.14.1/bin/hdfcoinc/pycbc_fit_sngls_binned#L99 
probably less confusing for users to have the same behaviour now.